rfcs/text/0025-container-metric-fields.md

```diff
@@ -13,19 +13,27 @@ Feel free to remove these comments as you go along.
 Stage 0: Provide a high level summary of the premise of these changes. Briefly describe the nature, purpose, and impact of the changes. ~2-5 sentences.
 -->
 
+<!--
+Stage 1: If the changes include field additions or modifications, please create a folder titled as the RFC number under rfcs/text/. This will be where proposed schema changes as standalone YAML files or extended example mappings and larger source documents will go as the RFC is iterated upon.
+-->
+
 This RFC proposes 6 additional container metric fields into ECS for monitoring
-container CPU, memory, disk and network performance using Metricbeat.
+container CPU, memory, disk and network performance using Metricbeat:
```

Comment:
```suggestion
container CPU, memory, disk and network performance:
```

Nit: Using the fields won't be limited to Metricbeat

Reply:
Good point! Done!